build.rs

```diff
@@ -119,17 +132,17 @@ fn write_testsuite_tests(
     testsuite: &str,
     strategy: &str,
 ) -> io::Result<()> {
-    let stemstr = path
+    let testname = path
         .file_stem()
         .expect("file_stem")
         .to_str()
         .expect("to_str");
 
     writeln!(out, "        #[test]")?;
-    if ignore(testsuite, stemstr, strategy) {
+    if ignore(testsuite, testname, strategy) {
```

Comment:
...therefore when we pass `testname` in to `ignore` here, it still has the hyphens, not the underscores.

Either we should replace all the hyphens early on, and then match `"multi_value"`, or we should replace the hyphens late and match `"multi-value"`.

Reply:
That's it! Thanks! I was trying to do the replacements early but at some point I either forgot about 136-139 or I rebased my changes away. Let's see if this latest push works.